operator/controllers/common/providers/keptnmetric/keptnmetric.go

```diff
@@ -5,8 +5,8 @@ import (
 	"fmt"
 
 	"github.com/go-logr/logr"
+	metricsv1alpha1 "github.com/keptn/lifecycle-toolkit/metrics-operator/api/v1alpha2"
```

Comment:
i'd suggest to rename the import alias `metricsv1alpha1` to something like `metricsapi`. so we don't need to adapt all occurrences of it whenever we change the imported path